src/components/info-tile/examples/info-tile-progress.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, h, State } from '@stencil/core';
+/**
+ * Displaying a progress bar instead of an icon
+ *
+ * It is possible to display a circular progress bar instead of an icon
+ * on the component. This can for instance help visualize how much of a set goal
+ * has been achieved, which together with the `value` will help users
+ * get a better overview of the provided data.
+ *
+ * :::tip
+ * It is possible to customize the progress bar's suffix, but it is
+ * set to display the percentage sign (**%**) by default.
```

Comment:
if I type this instead:
```md
set to display `%` by default.
```
Kompendium will crash and not display any DOCs for this example. @jgroth FYI